bubble_sort/Python3/mergeSort.py

```diff
@@ -0,0 +1,16 @@
+def bubbleSort(arr):
+    n = len(arr)
+ 
+    # Traverse through all array elements
+    for i in range(n):
+ 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpyaw51koo/bubble_sort/Python3/mergeSort.py
+++ b/tmp/tmpyaw51koo/bubble_sort/Python3/mergeSort.py
@@ -3,7 +3,7 @@
  
     # Traverse through all array elements
     for i in range(n):
- 
+
         # Last i elements are already in place
         for j in range(0, n-i-1):
  
```